solidity/dashboard/src/css/typography.less

```diff
@@ -101,24 +101,13 @@ a {
   color: @secondary-color;
   position: relative;
 
-  &.arrow-link {
+  &.external {
     &::after {
-      content: '';
-      position: absolute;
-      width: 1.25em;
-      height: 1.25em;
-      background-size: 1.25em 1.25em;
-      top: 50%;
-      margin-top: -0.625em;
-      right: calc(-1.25em - 2px);
-      background-image: url("../static/svg/link-arrow.svg");
-
-    }
-    
-    &.grey {
-      &::after {
-        background-image: url("../static/svg/link-arrow-grey.svg");
-      }
+      // north east arrow code
+      content: '\2197';
+      margin-left: 0.25rem;
+      // Avoid `text-decoration` under `::after` content
+      display: inline-block;
```

Comment:
Can't you just `text-decoration: none`? Also, do we _want_ the decoration not to cover the arrow? It's part of the hover hit area. This and the above change would also give us some consistency in how we handle things across our properties.

Last but not least: the approach I linked to above may be a good blanket approach here. No need to mark something with a class, you can match it based on the `href`.

Reply:
> Can't you just text-decoration: none? Also, do we want the decoration not to cover the arrow? It's part of the hover hit area. This and the above change would also give us some consistency in how we handle things across our properties.

As I can see on Figma we skip `text-decoration` under the arrow. Pls [see](https://www.figma.com/file/ARAMFLMOPal0JewjvlZGhX/Keep-Design-System?node-id=1097%3A0)

Reply:
>Last but not least: the approach I linked to above may be a good blanket approach here. No need to mark something with a class, you can match it based on the href.

Right, but we want an arrow only for external links.

Reply:
and we can't just `text-decoration: none` because 
> Note that text decorations are not propagated to floating and absolutely positioned descendants, nor to the contents of atomic inline-level descendants such as inline blocks and inline tables.

[doc](https://www.w3.org/TR/CSS21/text.html#propdef-text-decoration)

Reply:
Ah yes, I remember running into this at some point. You can't override a parent's text-decoration if you're operating on an inline element.